src/usb.c

```diff
@@ -262,6 +262,30 @@ int _stlink_usb_write_debug32(stlink_t *sl, uint32_t addr, uint32_t data) {
     return 0;
 }
 
+int _stlink_usb_get_rw_status(stlink_t *sl)
+{
+    if (sl->version.jtag_api == STLINK_JTAG_API_V1)
+        return -1;
+
+    unsigned char* const rdata = sl->q_buf;
+    struct stlink_libusb * const slu = sl->backend_data;
+    unsigned char* const cmd  = sl->c_buf;
+    int i, ret = 0;
```

Comment:
This 'ret' caused a build failure on 32bit systems

Reply:
Thx. So can we go for an int16_t here then?

Reply:
The answer obviously seems to be "yes". :laughing: